src/ucs/debug/assert.h

```diff
@@ -69,9 +69,9 @@ BEGIN_C_DECLS
 
 #define UCS_ENABLE_ASSERT 0
 
-#define ucs_bug(...)
-#define ucs_assert(...)
-#define ucs_assertv(...)
+#define ucs_bug(...)     do {} while(0)
+#define ucs_assert(...)  do {} while(0)
+#define ucs_assertv(...) do {} while(0)
```

Comment:
This is needed to silence cppcheck complaining about uninitialised value.
When built in release mode we used empty value for these macros. And when this macro is used in statement expression, then cppcheck complains about uninitialized value
```
#define ucs_alloca(_size) \
    ({ \
        ucs_assertv((_size) <= UCS_ALLOCA_MAX_SIZE, "alloca(%zu)", (size_t)(_size)); \
        alloca(_size); \
    })
```
unfolded as:
```
    ({
        ;
        alloca(_size);
    })
```
Despite this is valid statement expression, cppcheck does not like it due to redundant semicolon. So this change is needed to silence that warning